Description: Fix test for recent mocha
Author: Yadd <yadd@debian.org>
Forwarded: no
Last-Update: 2021-11-16

--- a/test/test.js
+++ b/test/test.js
@@ -98,7 +98,7 @@
         cryptoRandomBytes.seeded = undefined
       })
 
-      it('should still generate bytes when later seeded', function () {
+      it('should still generate bytes when later seeded', function (done) {
         cryptoRandomBytes.seeded = 1
 
         randomBytes(18, function (err, buf) {
@@ -109,7 +109,7 @@
         })
       })
 
-      it('should attempt generation three times', function () {
+      it('should attempt generation three times', function (done) {
         cryptoRandomBytes.seeded = 2
 
         randomBytes(18, function (err, buf) {
@@ -120,7 +120,7 @@
         })
       })
 
-      it('should error if never seeded', function () {
+      it('should error if never seeded', function (done) {
         cryptoRandomBytes.seeded = false
 
         randomBytes(18, function (err) {
